pkgs/development/compilers/graalvm/community-edition.nix

```diff
@@ -1,48 +1,60 @@
-{ stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath }:
+{ stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath, lib, darwin, openssl }:
 
 let
+  platform = if stdenv.isDarwin then "darwin-amd64" else "linux-amd64";
   common = javaVersion:
     let
+      javaVersionPlatform = "${javaVersion}-${platform}";
```

Comment:
```suggestion
      javaVersionPlatform = "${javaVersion}-${if stdenv.isDarwin then "darwin" else "linux"}-amd64";
```

Reply:
`javaVersion` is a parameter of the function `common`.  Strings "8" and "11" is passed from here:
https://github.com/NixOS/nixpkgs/blob/c5785e1d01c7f646aad6e266c84ca5001c8c3af9/pkgs/development/compilers/graalvm/community-edition.nix#L215-L216